noir-projects/aztec-nr/value-note/src/value_note.nr

```diff
@@ -13,7 +13,7 @@ use dep::aztec::{
     },
 };
 
-global VALUE_NOTE_LEN: u32 = 3; // 3 plus a header.
+pub global VALUE_NOTE_LEN: u32 = 3; // 3 plus a header.
```

Comment:
Warnings fix --> value was used in a test contract.

Reply:
Which test contract? Accessing this is an antipattern.

Reply:
I got confused and its only used within a create so I replaced `pub` with `pub(crate)` in https://github.com/AztecProtocol/aztec-packages/pull/11129/commits/d4485d382a57ddd1c353ce610e56223b72b0df46